LEGO1/lego/legoomni/include/legoracemap.h

```diff
@@ -0,0 +1,35 @@
+#ifndef LEGORACEMAP_H
+#define LEGORACEMAP_H
+
+#include "legoraceactor.h"
+
+/*
+	VTABLE: LEGO1 0x100d5510 LegoRaceActor
```

Comment:
3 classes have the same vtable address?

Reply:
Nope, All virtual classes should have completely different tables, even if nothing is overridden. Technically the compiler could coalesce them, but it doesn't. I'll doublecheck them all.